package.json

```diff
@@ -60,18 +60,18 @@
     "@ew-did-registry/proxyidentity": "0.7.1-alpha.795.0",
     "@nestjs/axios": "0.1.0",
     "@nestjs/bull": "^0.6.0",
-    "@nestjs/common": "^8.0.6",
-    "@nestjs/config": "^1.0.1",
-    "@nestjs/core": "^8.0.6",
+    "@nestjs/common": "^9.2.1",
+    "@nestjs/config": "^2.2.0",
+    "@nestjs/core": "^9.2.1",
     "@nestjs/event-emitter": "^1.0.0",
-    "@nestjs/graphql": "^8.0.2",
-    "@nestjs/jwt": "^8.0.0",
-    "@nestjs/passport": "^8.1.0",
-    "@nestjs/platform-express": "^8.0.6",
-    "@nestjs/schedule": "^1.0.1",
-    "@nestjs/swagger": "^5.0.9",
-    "@nestjs/terminus": "^8.0.1",
-    "@nestjs/typeorm": "^8.1.2",
+    "@nestjs/graphql": "^10.1.6",
```

Comment:
It's better to update dependencies through nx migrations: https://nx.dev/core-features/automate-updating-dependencies
By doing it manually it can make problems with migrations in future. Those migrations also updates code in apps/libs by removing deprecations or API changes.